client/me/connected-application-item/index.jsx

```diff
@@ -120,116 +118,100 @@ module.exports = React.createClass( {
 
 		if ( message ) {
 			return (
-				<li className="connected-application-item__connection-detail">
-					<strong className="connected-application-item__connection-detail-title">
+				<div>
+					<h2>
 						{ this.translate( 'Access Scope' ) }
 						{ this.renderAccessScopeBadge() }
-					</strong>
+					</h2>
 
-					<span className="connected-application-item__connection-detail-description" >
+					<p className="connected-application-item__connection-detail-description" >
 						{ message }
-					</span>
-				</li>
+					</p>
+				</div>
 			);
 		}
 	},
 
 	renderDetail: function() {
-		var connection = this.props.connection;
 		if ( this.props.isPlaceholder ) {
 			return;
 		}
 
 		return (
-			<div className="connected-application-item__content">
-				<p className="connected-application-item__content-description">
-					{ connection.description }
+			<div>
+				<h2>
+					{ this.translate( 'Application Website' ) }
+				</h2>
+				<p>
+					<a
+						href={ safeProtocolUrl( this.props.connection.URL ) }
+						onClick={ this.recordClickEvent( 'Connected Application Website Link' ) }
+						target="_blank"
+					>
+						{ safeProtocolUrl( this.props.connection.URL ) }
```

Comment:
This is another place where destructuring is nice because it takes out the distracting and overly-specific `this.props.`. In our render, we care more about a `URL` or a `connection.URL` than we do a `this.props.connection.URL`

```js
const { URL } = this.props.connection;

// or

const { connection: { URL } } = this.props;
```